Cargo.toml

```diff
@@ -26,8 +26,6 @@ members = [
     "variants/aws-k8s-1.30-nvidia",
     "variants/aws-k8s-1.31-nvidia",
     "variants/metal-dev",
-    "variants/metal-k8s-1.28",
```

Comment:
I think we should drop the corresponding settings-plugins for these variants (looks like I'd missed that in the 1.27 drop amidst the core kit shift 😅): https://github.com/bottlerocket-os/bottlerocket/blob/b681183a68201d50ba0bd408637789a597b86fa2/packages/settings-plugins/settings-plugins.spec#L98-L109

Reply:
That makes sense, I removed the defaults and plugins for metal-k8s.